pkg/plugin/plugin.go

```
@@ -225,6 +225,17 @@ func formatAlias(alias string, values map[string]interface{}) string {
 	return string(formattedAlias)
 }
 
+// renderField renders the value behind one {{ field }} placeholder of a legend
+// alias. Strings are passed through as they are so that quoting is preserved.
+func renderField(values map[string]interface{}, name string) string {
```

golangci-lint (2.12.2)

[error] func renderField is unused

(unused)